frontend/mobile/components/club-recruitment-info.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, Text, ColorName, Colors} from "@/app/(design-system)"
+import RecruitmentCycleIcon from '@/assets/svg/recruitment-cycle.svg';
+import RecruitmentStatusIcon from '@/assets/svg/recruiting-status.svg';
+import ApplicationTypeIcon from '@/assets/svg/application-type.svg';
+import { RecruitmentCycle, RecruitmentType } from "@generatesac/lib";
+import { SvgProps } from 'react-native-svg';
+import { ReactElement } from 'react';
+
+interface RecruitmentInfoProps {
+    color: ColorName,
+    recruitmentCycle: RecruitmentCycle,
+    recruitingType: RecruitmentType,
+    isRecruiting?: boolean,
+    recruitmentText: string,
+}
+
+interface RecruitmentItemProps {
+    icon: ReactElement<SvgProps>,
+    title: string,
+    text?: string | undefined,
+}
+
+const capitalize = (str: string) => {
+    if (!str) return '';
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+export const RecruitmentInfo = ({recruitmentText, color, recruitmentCycle, recruitingType, isRecruiting = false}: RecruitmentInfoProps) => {
+    const RecruitmentItem = ({icon, title, text}: RecruitmentItemProps) => {
```

Comment:
Piggy backing off the lint warning, define `RecruitmentItem` as it's own component in it's own file and then import it to use here